### PR TITLE
remove useless imports

### PR DESCRIPTION
**Pull request by runzhliu**
Obviously, there is currently no check for code styles including useless import, now I remove all of the useless imports.

### CODE REVIEW

**Comment by databricks-cla-assistant** [comment]
[CLA assistant check](https://cla.dev.databricks.com/delta-io/delta?pullRequest=12) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dev.databricks.com/delta-io/delta?pullRequest=12) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dev.databricks.com/check/delta-io/delta?pullRequest=12) it.</sub>

**Comment by brkyvz** [comment]
Closing and re-opening to trigger a test

**Comment by tdas** [comment]
Hey @runzhliu thank you for your PR. Give us a few days to polish up our infrastructure to properly test PRs, ensure style checks, etc. We will get back to this PR as soon as possible. Thanks for you patience.

**Comment by runzhliu** [comment]
Hi @tdas, happy to get ur reply, I'm very looking forward to the development of this project!

File: src/main/scala/org/apache/spark/sql/delta/metering/DeltaLogging.scala [hunk 1]
```diff
@@ -44,7 +44,7 @@ import org.apache.spark.sql.delta.util.JsonUtils
  *    that match "delta.ddl.%".
  *
  *  Underneath these functions use the standard usage log reporting defined in
- *  [[com.databricks.spark.util.UsageLogging]].
+ *  [[com.databricks.spark.util.DatabricksLogging]].
```
Comment (mukulmurthy) [inline src/main/scala/org/apache/spark/sql/delta/metering/DeltaLogging.scala]: This should not be changed.

**Pull request closed without merging**